_data/committers.yml

```diff
@@ -139,14 +159,14 @@
   role: PMC
   alias: tdunning
   affiliation: MapR
+- name: Todd Lipcon
+  role: PMC
+  alias: todd
+  affiliation: Google
 - name: Uwe L. Korn
   role: PMC
   alias: uwe
   affiliation: Quantco
-- name: Ben Kietzman
-  role: PMC
-  alias: bkietz
-  affiliation: Ursa Computing
```

Comment:
Nit, but could you add a blank line between the PMCs and the committers here?

Reply:
Done